coremltools/converters/mil/mil/ops/defs/recurrent.py

```diff
@@ -9,6 +9,80 @@
 
 @register_op(doc_str="TODO")
 class gru(Operation):
+    """
+    Gated recurrent unit (GRU).
+    
+    .. math::
+       r_t = recurrent_activation(W_{ir} x_t + b_{ir} + W_{hr} h_(t−1) + b_{hr})
+    
+    .. math::
+       z_t = recurrent_activation(W_{iz} x_t + b_{iz} + W_{hz} h_(t−1) + b_{hz})
+    
+    .. math::
+       o_t = activation(W_{io} x_t + b_{io} + r_t * (W_{ho} h_(t−1) + b_{ho}))
+    
+    .. math::
+       h_t = (1 − z_t) * o_t + z_t * h_{(t−1)}
+    
+    Where:
+
+    * ``W_{ir}``, ``W_{iz}``, and `` W_{io}`` state input-hidden weight for reset, update
+      and output gate, respectively.
+    * Similar to the above, ``W_{h[r/z/o]}`` states hidden-hidden / recurrent weights.
+    * ``h_t``  is the hidden state at time ``t``.
+    * ``x_t`` is the input at time ``t``.
+    * ``h_(t-1)`` is the hidden state of the layer at time ``t-1`` or the initial
+      hidden state at time ``0``.
+    * ``r_t``, ``z_t``, and ``n_t`` are the reset, update, and new gates, respectively.
+    * ``*`` is elementwise product.
+    
+    Parameters
+    ----------
+    x: <s, b, I, T> (Required)
+        * ``s`` is the sequence length, ``b`` is the batch size, and ``I`` is the
+          input dimension.
+    
+    initial_h: <b, H, T> (Required)
+        * ``H`` denotes hidden size.
+    
+    weight: const<I+H, 3*H, T> (Required) - Weight matrix
+        * ``weight[:I] =  [W_{iz} | W_{ir} | W_{in}]`` where ``[a|b]`` denotes column
+          concatenation and ``[a, b]`` denotes row concatenation. ``W_{iz}``,
+          ``W_{ir}``, and ``W_{in}`` have shape ``(I, H)``.
+        * ``weight[I:] =  [W_{hz} | W_{hr} | W_{hn}]``: ``W_{hz}``, ``W_{hr}``, and
+          ``W_{hn}`` have shape ``(H, H)``.
+    
+    bias: const<2, 3*H, T> (Optional) [Default all 0s]
+        * ``bias[0]`` and ``bias[1]`` are input-hidden and hidden-hidden
+          bias, respectively.
+        * ``3*H`` are biases for ``[``.
```

Comment:
This sentence is incomplete

Reply:
Sentence is incomplete in Quip doc.

Reply:
Sentence is now: ``3*H`` are biases for ``[b_{ir} + b_{hr}, b_{iz} + b_{hz}, b_{io} + b_{ho}]``.